src/raw-transaction-status.ts

```diff
@@ -1,6 +1,43 @@
+import { TransactionStatus } from './generated/web/legacy/transaction_status_pb'
+import { GetTxResponse } from './generated/web/rpc/v1/tx_pb'
+
 /** Abstraction around raw Transaction Status for compatibility. */
-export default interface RawTransactionStatus {
-  getValidated(): boolean
-  getTransactionStatusCode(): string
-  getLastLedgerSequence(): number
+export default class RawTransactionStatus {
```

Comment:
Was this class/interface part of the public API of `xpring-js`? Is this a breaking change?

Reply:
No. It's not exported in `index.js` and it's also not available in the public API of anything that is. 
 This is just an internal construct. 

When the legacy protocol buffers go away, so will this class.